FeedBuilder/frmMain.cs

```diff
@@ -81,18 +83,9 @@ private void frmMain_Load(Object sender, EventArgs e)
 			if (!_argParser.ShowGui) Close();
 		}
 
-		private void InitializeFormSettings()
+		private async Task InitializeFormSettings()

```

Comment:
Marking the method as async does not make it run async per default.

All places that calls InitializeFormSettings() also need to be changed.

**Example:**

	public class AsyncTestClass
	{
		// Output:
		// Initialize() done
		// Run() done
		public void Run()
		{
			Initialize();
			Console.WriteLine("Run() done");
		}

		// Output:
		// Run() done
		// Initialize() done
		public void RunAsync()
		{
			Task.Run(Initialize);
			Console.WriteLine("Run() done");
		}


		private async Task Initialize()
		{
			Thread.Sleep(2000);
			Console.WriteLine("Initialize() done");
		}
	}